config/initializers/brewerydb.rb

```diff
@@ -0,0 +1,4 @@
+$brewery_db = BreweryDB::Client.new do |config|
+  config.api_key = ENV['BREWERYDB_APIKEY']
```

Comment:
Use `ENV.fetch['BREWERYDB_APIKEY']` with this you can avoid missing keys when you deploy this to production, check this http://michalorman.com/2015/02/fetch-your-environment-variables/